misc/server_log.yml

```
@@ -2,6 +2,7 @@
 formatters:
   core_console_formatter:
     # Console formatter with coloring
+    # This formatter also does indentation of multi-line log message, so all lines are put nicely underneath each other.
     (): inmanta.logging.MultiLineFormatter
     fmt: '%(log_color)s%(name)-25s%(levelname)-8s%(reset)s%(blue)s%(message)s'
```

Looking at `misc/server_lsm_log.yml`, I would expect this here as well if `--timed-logs` option is passed
```suggestion
    fmt: '%(asctime)s %(log_color)s%(name)-25s%(levelname)-8s%(reset)s%(blue)s%(message)s'
```